package-lock.json

```diff
@@ -4,6 +4,24 @@
     "lockfileVersion": 1,
     "requires": true,
     "dependencies": {
+        "@azure/cosmos": {
```

Comment:
I am not sure why this file had changes this time. I have been following the "rm -rf node_modules/", "npm cache clean --force" and "npm i" steps to make sure this file doesn't have unnecessary changes

Reply:
You only need to do that if you see it changing "https" to "http". It's related to a bug in older versions of npm, so it really shouldn't happen going forward since I believe your npm is up to date now